sdk/messaging/eventgrid/azsystemevents/models.go

```diff
@@ -1,438 +1,14 @@
-//go:build go1.18
-// +build go1.18
-
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
-// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
-// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+// Code generated by Microsoft (R) Go Code Generator. DO NOT EDIT.
 
 package azsystemevents
 
 import "time"
 
-// APICenterAPIDefinitionAddedEventData - Schema of the data property of an EventGridEvent for a Microsoft.ApiCenter.ApiDefinitionAdded
```

Comment:
All of these breaking changes are expected?

Reply:
TBH, I'm a bit conflicted. They're "expected" in that that's the way the models really are on the wire, but "unexpected" in that the arch board did decide on better names. I think I'm going to just apply the renames to bring them back to what was decided so we match what others are shipping.

Reply:
I've reapplied the old fixes, so this is no longer a problem.